README.md

```diff
@@ -32,26 +32,27 @@ You also need to mount session related endpoints to your app, in your `config/ro
 mount ArtsyAuth::Engine => '/'
 ```
 
-In order to force authentication, you need to change your `ApplicationController` to inherit from ` ArtsyAuth::ApplicationController`, you also need to add (override) `authorize?` method there which gets a token and in your app you need to define how do you authorize that token, for example:
+In order to force authentication, you need to include 'ArtsyAuth::Authenticated' in your controller, you also need to add (override) `authorized_artsy_token?` method there which gets a token and in your app you need to define how do you authorize that token, for example:
 ```ruby
 class ApplicationController < ArtsyAuth::ApplicationController
```

Comment:
This is still extending `ArtsyAuth`. From the text it sound like this isn't the case anymore. Is this correct?